docs/static/releasenotes.asciidoc

```diff
@@ -15,8 +15,15 @@ endif::include-xpack[]
 [[logstash-6-0-0-rc1]]
 === Logstash 6.0.0-rc1 Release Notes
 
-Placeholder for RC1 release notes.
-
+* https://github.com/elastic/logstash/pull/8318[Added all commercially supported plugins to bundled plugins]
+* existing `JAVA_OPTS` env var is now ignored for consistency with Elasticsearch
```

Comment:
Any instructions for users who are impacted by this change? Maybe something like: "Use LS_JAVA_OPTS instead or set java flags in `config/jvm.options`"